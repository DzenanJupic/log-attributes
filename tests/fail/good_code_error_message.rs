#[log_attributes::log(info, "{fn} was executed")]
fn good_code_error_message() -> u32 {
    let _: () = 2;
}

fn main() {}
