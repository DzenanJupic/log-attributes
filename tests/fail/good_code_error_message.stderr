error[E0308]: mismatched types
 --> tests/fail/good_code_error_message.rs:3:17
  |
3 |     let _: () = 2;
  |            --   ^ expected `()`, found integer
  |            |
  |            expected due to this

error[E0308]: mismatched types
 --> tests/fail/good_code_error_message.rs:2:37
  |
2 |   fn good_code_error_message() -> u32 {
  |  _________________________________---_^
  | |                                 |
  | |                                 expected `u32` because of return type
3 | |     let _: () = 2;
4 | | }
  | |_^ expected `u32`, found `()`
